u_parrots_test.go

```diff
@@ -204,15 +204,14 @@ func checkUTLSFingerPrintClientHello(t *testing.T, clientHelloID ClientHelloID,
 
 func TestUTLSFingerprintClientHello(t *testing.T) {
 	clientHellosToTest := []ClientHelloID{
-		HelloChrome_58, HelloChrome_70, HelloChrome_83, HelloFirefox_55, HelloFirefox_63, HelloIOS_11_1, HelloIOS_12_1, HelloRandomized, HelloRandomizedALPN, HelloRandomizedNoALPN}
-
-	serverNames := []string{"foobar"}
+		HelloChrome_58, HelloChrome_70, HelloChrome_83, HelloFirefox_55, HelloFirefox_63,
+		HelloEdge_85, HelloExplorer_11, Hello360_7_5, HelloQQ_10_6, HelloIOS_11_1, HelloIOS_12_1,
+		HelloRandomized, HelloRandomizedALPN, HelloRandomizedNoALPN,
+	}
```

Comment:
I added the new hello IDs to the fingerprint test.  They fail now, but pass if you merge in https://github.com/getlantern/utls/pull/9.